tests/cupy_tests/creation_tests/test_basic.py

```diff
@@ -231,3 +231,115 @@ def test_full_default_dtype_cpu_input(self, xp, dtype):
     def test_full_like(self, xp, dtype, order):
         a = xp.ndarray((2, 3, 4), dtype=dtype)
         return xp.full_like(a, 1, order=order)
+
+
+@testing.parameterize(
+    *testing.product({
+        'shape': [(4, 2), (4, 2, 3), (5, 4, 2, 3)],
+    })
+)
+@testing.gpu
+class TestBasicReshape(unittest.TestCase):
+    @testing.with_requires('numpy>=1.17.0')
```

Comment:
Please add test cases that does not depend on NumPy 1.17.0.

Reply:
Done, I added new tests to cover about the same cases, only without comparing to NumPy results.